#[test]
fn test_error() {
    panic!("Make this test fail")
}